iothub/device/samples/convention-based-samples/readme.md

```diff
@@ -12,41 +12,557 @@ products:
 urlFragment: azure-iot-pnp-device-samples-for-csharp-net
 ---
 
-# IoT Plug And Play device samples
+# IoT Plug And Play (PnP) device/ module APIs
```

Comment:
I noticed you have a space after slashes in this document. Is this a habit of typing :)? Or is it really the convention?

Reply:
oh, that is a habit; is that incorrect?

Reply:
I did some research. Since you're indicating an 'or', no space is needed. Here are examples from this site:
"When leaving the classroom, the teacher noticed that a student had left his/her backpack."
"College freshmen should bring a mattress and/or cot to sleep on during orientation."

Reference: https://www.grammarly.com/blog/slash/